src/core/internationalization.js

```diff
@@ -127,6 +129,7 @@ export let translator = (key, values) => {
  * Set up our translation function, with loaded languages
  */
 export const initialize = () => {
+  let latestMinorVersionPath = 'https://cdn.jsdelivr.net/npm/p5@' + constants.VERSION.replace(/^(\d+\.\d+)\.\d+.*$/, '$1');
```

Comment:
Hi @ksen0 , @limzykenneth just a thought here.
Will it be a better idea  to separate out the base `CDN` URL here and the version parsing logic for better readability and reusability too, and then compose the final `loadPath`. I was thinking to encapsulate it like below:
```js

const getCDNPath = (() => {
  const base = 'https://cdn.jsdelivr.net/npm/p5@';
  const version = constants.VERSION.replace(/^(\d+\.\d+)\.\d+.*$/, '$1');
  return `${base}${version}`;
})();
```

Reply:
I'm not sure it's worth to split it out in this way, especially also encapsulating it in a function, if needed, just a string interpolation should suffice as well.
```js
`https://cdn.jsdelivr.net/npm/p5@${constants.VERSION.replace(/^(\d+\.\d+)\.\d+.*$/, '$1')}`
```

Reply:
just for the sake of better separation of the base and version following the composition will benefit in the better readability, I thought.